src/webview/webkitgtk/mod.rs

```diff
@@ -77,15 +78,6 @@ impl InnerWebView {
     let webview = Rc::new(webview);
     let w = window_rc.clone();
     let ipc_handler = attributes.ipc_handler.take();
-    let manager = web_context.manager();
-
-    // Use the window hash as the script handler name to prevent from conflict when sharing same
-    // web context.
-    let window_hash = {
-      let mut hasher = DefaultHasher::new();
-      w.id().hash(&mut hasher);
-      hasher.finish().to_string()
-    };
 
```

Comment:
```suggestion
    // Use the window hash as the script handler name to prevent from conflict when sharing same
    // web context.
    let window_hash = {
      let mut hasher = DefaultHasher::new();
      w.id().hash(&mut hasher);
      hasher.finish().to_string()
    };
```